torch/csrc/jit/script/compiler.cpp

```diff
@@ -288,18 +354,10 @@ static bool isTensorSubtype(Value* v) {
   return v->type()->isSubtypeOf(*DynamicType::get());
 }
 
-static bool isNumberSubtype(const Value* v) {
-  return v->type()->isSubtypeOf(*NumberType::get());
-}
-
-static bool isNumberSubtype(const TypePtr& type) {
-  return type->isSubtypeOf(*NumberType::get());
-}
-
 at::optional<std::vector<int64_t>> getIntListAttribute(at::optional<int32_t> N, Value* input) {
-  auto list = constant_as<std::vector<int64_t>>(input);
+  auto list = constant_as<Shared<jit::IntList>>(input);
   if(list)
-    return list;
+    return std::vector<int64_t>((*list)->elements().begin(), (*list)->elements().end());
```

Comment:
Can we just add an `operator std::vector<int64_>` to `jit::IntList` (`at::IntList` has it)?

Reply:
I added the conversion to at::ArrayRef<Elem> for constant lists. This makes sure you are still explicit when you are copying the list but makes this expression only:
`std::vector<int64_t>(*list)`